webpack.config.js

```diff
@@ -1,26 +1,23 @@
 const path = require('path');
 
-const SRC_DIR = path.join(__dirname, '/client/');
-const DIST_DIR = path.join(__dirname, '/public/');
 
 module.exports = {
   mode: 'development',
-  entry: `${SRC_DIR}/index.jsx`,
+  entry: './client/index.jsx',
   output: {
-    filename: 'app.js',
-    path: DIST_DIR,
+    filename: 'bundle.js',
+    path: path.join(__dirname, 'public'),
   },
   module: {
-    rules: [
-      {
-        test: /\.jsx?/,
-        include: SRC_DIR,
+    rules: [{
+      test: /\.jsx?/,
```

Comment:
Don't forget to allow webpack to bundle normal JS. I'm not sure if it's necessary but it's something thats been done in all our repos using webpack up till now.  `test: /\.(js|jsx)$/`